docs/example/saxml/leader-worker-set.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: leaderworkerset.x-k8s.io/v1
+kind: LeaderWorkerSet
+metadata:
+  name: saxml-multi-host
+  annotations:
+    leaderworkerset.sigs.k8s.io/exclusive-topology: cloud.google.com/gke-nodepool
+spec:
+  replicas: 2
+  leaderWorkerTemplate:
+    size: 8
+    restartPolicy: RecreateGroupOnPodRestart
+    leaderTemplate:
+      metadata:
+        labels:
+            app: leader-pod
+      spec:
+        serviceAccountName: sax-sa
+        nodeSelector:
+          cloud.google.com/gke-tpu-topology: 4x8
+          cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
+        containers:
+        - name: sax-admin-server
+          image: us-docker.pkg.dev/cloud-tpu-images/inference/sax-admin-server:nightly
+          securityContext:
+            privileged: true
+          ports:
+          - containerPort: 10000
+          env:
+          - name: SAX_REPLICA_NAME
+            valueFrom:
+              fieldRef: 
+                fieldPath: metadata.name 
+          - name: SAX_GCS_BUCKET
+            value: BUCKET_NAME
+        - name: sax-model-server
+          image: us-docker.pkg.dev/cloud-tpu-images/inference/sax-model-server:nightly
+          args: ["--port=10001", "--platform_chip=tpuv5e"]
+          ports:
+          - containerPort: 10001
+          - containerPort: 8471
+          securityContext:
+            privileged: true
+          env:
+          - name: SAX_REPLICA_NAME
+            valueFrom:
+              fieldRef: 
+                fieldPath: metadata.name 
+          - name: SAX_GCS_BUCKET
+            value: BUCKET_NAME
+          resources:
+            requests:
+              google.com/tpu: 4
+            limits:
+              google.com/tpu: 4
```

Comment:
We can simplify the yaml quite a bit if we don't make the admin a worker too, https://github.com/kubernetes-sigs/lws/pull/61 will allow us to do that. Can you help test it with SaxML?

Reply:
Test it out with #61 changes applied, simplified yaml worked. Updated the yaml